style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main-container {
+  color: whitesmoke;
+  background-color: #1c1a19;
+  min-width: 375px;
+  width: 100%;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #3e3c3c;
+  padding-left: 24px;
+  padding-right: 24px;
+  height: 92px;
+  align-items: center;
+}
+
+.me {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 22px;
+  width: 80px;
+  left: 142px;
+}
+
+.initial {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  min-height: 592px;
+  height: 100vh;
+  min-width: 375px;
+  width: 100%;
+  padding-left: 24px;
+  padding-right: 24px;
+  background: url("images/illustration-header-mobile.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+.information {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0.4px;
+  font-size: 58px;
+  line-height: 70px;
+  color: #f55800;
+}
+
+h2 {
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: left;
+}
```

Comment:
- [OPTIONAL] Kindly remove `Italic` on the subtitle in order to look like the provided Figma design. :wink: 
- [OPTIONAL] Kindly make sure to use the same text color.

     - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/198531620-d75dbc28-7950-4d83-9788-5752a10f1833.png)


     - ### Figma design
     ![image](https://user-images.githubusercontent.com/87197412/198531838-b7f6c876-37e8-439b-808d-6a08a0c0ddde.png)

Reply:
Great, I will make these changes !!